src/nswcaselaw/decision.py

```diff
@@ -346,22 +347,42 @@ class NewScScraper(Scraper):
         "catchwords": "Catchwords",
         "legislationCited": "Legislation Cited",
         "casesCited": "Cases Cited",
-        "parties": "Parties",
+        "textCited": "Texts Cited:",
         "category": "Category",
+        "parties": "Parties",
         "fileNumber": "File Number",
         "representation": "Representation",
     }
 
     def _scrape_metadata(self):
-        """Scrape the core metadata from the soup object"""
+        """Scrape the core metadata from the soup object.
+        Store Decision under appeal as e.g.:
+                 self._values['decisionUnderAppeal'] = {
+                     'Court or tribunal:': ['Supreme Court of New South Wales'],
+                     'Jurisdiction:': ['Equity Division'],
+                     'Date of Decision:': ['17 April 2018'],
+                     'Before:': ['Pembroke J'],
+                     'File Number(s):': ['2017/00120274']}"""
+
+        decisionUnderAppeal = {}
         for dt in self._soup.find_all("dt"):
-            dd = dt.find_next_sibling("dd")
             header = dt.string
-            paras = dd.find_all("p")
-            if paras:
-                self._raw[header] = [self._strings(p) for p in paras]
+            # if dt is Decision under appeal, read it into self._values
+            if header.strip() == "Decision under appeal":
+                div = dt.find_next_sibling("div")
+                for div_dt in div.find_all("dt"):
+                    decisionUnderAppeal[div_dt.string.strip()] = []
+                    for s in div_dt.find_next_sibling("dd").stripped_strings:
+                        decisionUnderAppeal[div_dt.string.strip()].append(s)
+                break
+
             else:
-                self._raw[header] = self._strings(dd)
+                dd = dt.find_next_sibling("dd")
+                paras = dd.find_all("p")
+                if paras:
+                    self._raw[header] = [self._strings(p) for p in paras]
+                else:
+                    self._raw[header] = self._strings(dd)
```

Comment:
There's no logic for extracting the decision-under-appeal data in the OldScScraper - is that because the old-style cases don't have the data?

Reply:
Yes, the old ones doesn't have that section